package.json

```
@@ -43,7 +43,7 @@
 		]
 	},
 	"lint-staged": {
-		"*.js,*.ts": "eslint --fix"
+		"*.js|*.ts": "eslint --fix"
```

if you want to go heavy...
```suggestion
		"*.{j,t}s?(x)": "eslint --fix"
```